typescript/types-backend/scripts/generate-openapi.ts

```diff
@@ -44,6 +46,24 @@ async function generate_types(appName: string) {
 
   console.log(`Outputting to ${outputPath}`);
   fs.writeFileSync(outputPath, output);
+
+  const header = `// This file is generated automatically. Do not edit.
+`;
+  let newFileContent = `${header}import * as openapi from "./${appName}-openapi";
+type schemas = openapi.components["schemas"];
+export { openapi };
+
+`;
```

Comment:
Tenker disse kan slås sammen
```suggestion
  const header = `// This file is generated automatically. Do not edit.
import * as openapi from "./${appName}-openapi";
type schemas = openapi.components["schemas"];
export { openapi };

`;
```